processes/release_processes/README.MD

```diff
@@ -7,11 +7,16 @@ See our "updating the process" [process](../README.MD)!
 
 ## Core Process: How to Publish and Release Cromwell
 
-If you're the lucky "Release Rotation Engineer" this time, you should do the following three things:
-
-First: Run the publish script to create a new version of Cromwell.
-Second: Run through the "How to Release Cromwell into Firecloud" process.
-Third: Run through the "How to Deploy Cromwell in CAAS prod" process.
+If you're the lucky "Release Rotation Engineer" this time, you should do the following four things:
+
+1. Check that all most recent executions of Cromwell's Jenkins tests have succeeded, and that all those executions were 
+against the most recent commit in "develop" branch. There are 2 groups of Jenkins tests which have to be checked (you 
+will need to be on the VPN to open the following links):
+    1. [Tests for various backends supported by Cromwell](https://gotc-jenkins.dsp-techops.broadinstitute.org/view/cromwell-cron/job/cromwell-cron-parent/)
```

Comment:
This link gives me a 404. Does it need to be shared?

Reply:
Are you using VPN?

Reply:
Yes